### PR TITLE
update SSO content

### PR DESCRIPTION
**Pull request by AlexicaWright**

### CODE REVIEW

File: docs/modules/ROOT/pages/operations/browser-sso.adoc [hunk 1]
```diff
@@ -27,102 +27,109 @@ It is strongly advised to use PKCE to ensure security.
...
-			}
-		}
-	]
-}
-----
-+
 [IMPORTANT]
 ====
-`redirect_uri` MUST match exactly the redirect_uri specified in the IdP.
+Security information should always be exchanged use encrypted transport, and therefore HTTPS should be used.
```
Comment (gregoryking) [inline docs/modules/ROOT/pages/operations/browser-sso.adoc:32]: typo: use

Comment (AlexicaWright) [reply docs/modules/ROOT/pages/operations/browser-sso.adoc:32]: How about :
_Security information should always be exchanged with encrypted transport, and therefore HTTPS should be used._?

File: docs/modules/ROOT/pages/operations/browser-sso.adoc [hunk 2]
```diff
@@ -1,21 +1,21 @@
...
-Your organization needs an existing compatible SSO provider (local or external) already configured.
+Neo4j Browser provides support for Single Sign-On (SSO) providers under specific Enterprise environment configurations when used with (self-hosted) Neo4j Server v4.4 and later.
+Additionally, you need a compatible SSO provider (local or external) already configured.
+For more information on the configuration, see link:https://neo4j.com/docs/operations-manual/current/authentication-authorization/sso-integration/#auth-sso-parameters[Operations Manual -> OIDC configuration settings^].
 
 [NOTE]
 ====
-Single Sign-On is _only_ available with the Enterprise Edition of Neo4j and requires engagement from Neo4j Professional Services to configure.
-General availability of SSO for Enterprise customers that does not require Professional Services' involvement is planned in a future release.
+For deployments other than with self-hosted Neo4j Server v4.4+, SSO can be configured with a separate OAuth plugin which requires engagement from Professional Services.
```
Comment (gregoryking) [inline docs/modules/ROOT/pages/operations/browser-sso.adoc:13]: Don't have anything to add to this myself other than to triple check with PS they are happy how this is presented. Mainly to ensure we don't drive too many people to enquire about the pre 4.4 SSO solution.

File: docs/modules/ROOT/pages/operations/browser-sso.adoc [hunk 3]
```diff
@@ -1,21 +1,21 @@
...
+[role=enterprise-edition]
 [[browser-sso]]
 = Browser Single Sign-On
 
-Neo4j Browser provides basic support for some Single Sign-On (SSO) providers under specific Enterprise environment configurations.
-Contact Neo4j Professional Services for additional assistance (see link:https://neo4j.com/professional-services/[_https://neo4j.com/professional-services/_^] for more information about Professional Services).
-Your organization needs an existing compatible SSO provider (local or external) already configured.
+Neo4j Browser provides support for Single Sign-On (SSO) providers under specific Enterprise environment configurations when used with (self-hosted) Neo4j Server v4.4 and later.
+Additionally, you need a compatible SSO provider (local or external) already configured.
+For more information on the configuration, see link:https://neo4j.com/docs/operations-manual/current/authentication-authorization/sso-integration/#auth-sso-parameters[Operations Manual -> OIDC configuration settings^].
```
Comment (jharris4) [inline docs/modules/ROOT/pages/operations/browser-sso.adoc:9]: This link seems broken to me, is that just because it hasn't been published yet?

Comment (AlexicaWright) [reply docs/modules/ROOT/pages/operations/browser-sso.adoc:9]: Correct, it'll work once the 4.4 docs are published.

File: docs/modules/ROOT/pages/operations/browser-sso.adoc [hunk 4]
```diff
@@ -27,102 +27,109 @@ It is strongly advised to use PKCE to ensure security.
...
+Security information should always be exchanged with encrypted transport, and therefore HTTPS should be used.
+Mixed HTTP/HTTPS flows for single sign-on are not supported.
 ====
 
+Once your SSO provider is configured, you need to configure Neo4j to use OpenID Connect.
+This is done by updating the link:https://neo4j.com/docs/operations-manual/current/configuration/neo4j-conf/#neo4j-conf[`neo4j.conf`^] file according to the instructions in link:https://neo4j.com/docs/operations-manual/current/authentication-authorization/sso-integration/#auth-sso-configure-sso[Operations Manual -> Configure Neo4j to use OpenId Connect^].
+
+Make sure to avoid duplicate entries in the `neo4j.conf` file.
+
+// Optionally, you may set extra logging for the OAuth2 plugin with these settings in the `neo4j.conf` file.
```
Comment (tbwiss) [inline docs/modules/ROOT/pages/operations/browser-sso.adoc:41]: Maybe check my comment for this part here on the PR in the neo4j-insight repo: https://github.com/neo-technology/neo4j-insight/pull/1896#discussion_r747585254

File: docs/modules/ROOT/pages/operations/browser-sso.adoc [hunk 5]
```diff
@@ -27,102 +27,123 @@ It is strongly advised to use PKCE to ensure security.
...
+
+Deployments that use an earlier version of Neo4j require a separate OAuth plugin and assistance from Professional Services, as mentioned previously.
+However, in such cases, the identity providers can be specified by a URL parameter `discoveryURL` that defines a URL to a json file containing the SSO providers.
+Example for Browser: `\https://<browser-server-host>:<http-port>?discoveryURL=https://webhost.com/public/discovery.json`
+
+[IMPORTANT]
+====
+Browser will prioritize in the following order, in case of conflicting data:
+. `neo4j.conf`
+. `discoveryURL`
```
Comment (OskarDamkjaer) [inline docs/modules/ROOT/pages/operations/browser-sso.adoc:52]: What data gets prioritized is a bit more complicated than this, we check a total of five sources and prioritize them in the following order:
- neo4j.conf data from the server entered in the :server connect form 
- neo4j.conf data from the server located at `connectURL` url param
- discoveryURL data
- neo4j.conf data from the server previously connected to (if any) 
- neo4j.conf data from the server that hosts browser

All data will be merged, given that the "host" field matches. Not sure what the best way to present this is, for the end user what it comes down to is -> point neo4j browser towards a 4.4 server and it'll use the config from that server. If you want to use some other config, use `discoveryURL`

**Review by gregoryking — APPROVED** [review]

**Pull request merged**